list.html

```diff
@@ -19,17 +19,6 @@
 
 <body>
     <h1>My Lists</h1>
-
-    <section class="nav">
-        <a class="nav-link" id="movie-link" href="#movie-section">
-            <i class="large material-icons">movie</i>
-            <p>My Movies</p>
-        </a>
-    
-        <a class="nav-link" id="tv-link" href="#tv-section">
-            <i class="large material-icons">tv</i>
-            <p>My TV</p>
-        </a>
```

Comment:
Why did we remove those?

Reply:
i thought we werent having a tv section since were only doing movies